exercises/simple-cipher/example.js

```diff
@@ -17,14 +17,14 @@ function randomUpTo(n) {
 module.exports = function (userDefinedKey) {
   var key;
 
-  function addEncodedCharacter(character, index, array) {
+  function addEncodedCharacter(character, index) {
     /* jshint validthis:true */
```

Comment:
Hmm...I missed this the first time around.  We should remove these jshint directives.  I'm not sure if they were merged inadvertently or are artifacts of an older linting setup, but they don't make sense anymore.